website/.env.sample

```diff
@@ -0,0 +1,21 @@
+# These environment variables are required so that the Firebase Admin SDK (server side) connects to the local emulator.
```

Comment:
I would name this `.env.local.sample` and only have entries that are not in the `.env.development` file (the mailchimp stuff..)